community/bolt/src/test/java/org/neo4j/bolt/v1/messaging/BoltRequestMessageWriter.java

```diff
@@ -58,7 +58,7 @@ public void onInit( String clientName, Map<String,Object> credentials ) throws I
     {
         packer.packStructHeader( 1, INIT.signature() );
```

Comment:
This is actually wrong but it is an error that has propagated to all drivers. The init struct has size 2 and not size 1.

Reply:
True :)
We do not really use struct size for predefined messages like INIT or RUN. It is kind of ignored [here](https://github.com/neo4j/neo4j/blob/3.4.0-alpha04/community/bolt/src/main/java/org/neo4j/bolt/v1/messaging/BoltRequestMessageReader.java#L58). Code decides how to unpack based on the struct signature. Worth fixing anyway, I'll create cards.